src/Service/Bulk/BulkCreateUsersService.php

```diff
@@ -33,70 +35,77 @@
 use OAT\SimpleRoster\Repository\Criteria\FindUserCriteria;
 use OAT\SimpleRoster\Repository\UserRepository;
 use OAT\SimpleRoster\Storage\UserGenerator\StorageInterface;
-use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
+use Symfony\Component\PasswordHasher\Hasher\UserPasswordHasherInterface;
```

Comment:
Does it use relevant hash algorithm? In other parts of project the deprecated is in use.

Reply:
Last time I got issues with login logic with other password encoder.

Reply:
`Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface` is deprecated
`/api/v1/auth/token` works fine with fresh generated users

Reply:
But what for existed user? Or we don't care?

Reply:
have you tried it? :)
I've changed it for encoder only. decoder remains the same, so it obviously should work with existing users.
but yes. it works for both new and old users